packages/__docs__/src/index.js

```diff
@@ -31,7 +31,9 @@ import { InstUISettingsProvider } from '@instructure/emotion'
 import '../globals'
 
 createRoot(document.getElementById('app')).render(
-  <InstUISettingsProvider theme={canvas}>
-    <App />
-  </InstUISettingsProvider>
+  <React.StrictMode>
+    <InstUISettingsProvider theme={canvas}>
+      <App />
+    </InstUISettingsProvider>
+  </React.StrictMode>
```

Comment:
Although the docs will be much slower in dev mode (because of the double mounting of `React.StrictMode`), we will able to catch potential bugs much sooner.